var/spack/repos/builtin/packages/tinyxml/package.py

```diff
@@ -23,16 +23,17 @@
 # Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
 ##############################################################################
 from spack import *
+from shutil import copyfile
 import os.path
 
 
 class Tinyxml(CMakePackage):
     """Simple, small, efficient, C++ XML parser"""
 
     homepage = "http://grinninglizard.com/tinyxml/"
-    url = "https://sourceforge.net/projects/tinyxml/files/tinyxml/2.6.2/tinyxml_2_6_2.tar.gz"
+    url = "https://downloads.sourceforge.net/project/tinyxml/tinyxml/2.6.2/tinyxml_2_6_2.tar.gz"
 
-    version('2.6.2', 'cba3f50dd657cb1434674a03b21394df9913d764')
+    version('2.6.2', 'cba3f50dd657cb1434674a03b21394df9913d764', url=url)
```

Comment:
It's probably better to use a `url_for_version`. Otherwise you'll have to update this for every new version.
```python
def url_for_version(self, version):
    url = "https://downloads.sourceforge.net/project/tinyxml/tinyxml/{0}/tinyxml_{1}.tar.gz"
    return url.format(version.dotted, version.underscored)
```